deploy/cr-minimal.yaml

```diff
@@ -11,7 +11,7 @@ spec:
     schedule: "0 4 * * *"
   pxc:
     size: 1
-    image: perconalab/percona-xtradb-cluster-operator:main-pxc8.0
+    image: percona/percona-xtradb-cluster-operator:1.13.0
```

Comment:
This needs to be database image (`percona/percona-xtradb-cluster:8.0.35-27.1`), you have put operator image here.

Reply:
done